packages/create-request-form/src/stores/requestNetworkStore.ts

```diff
@@ -0,0 +1,39 @@
+import { sepolia } from "viem/chains";
+import { createWalletClient, custom } from "viem";
+import { writable, type Writable } from "svelte/store";
+import { RequestNetwork } from "@requestnetwork/request-client.js";
+import { Web3SignatureProvider } from "@requestnetwork/web3-signature";
+
+export const requestNetworkStore: Writable<{
+  requestNetwork: RequestNetwork;
+  signer: string;
+} | null> = writable(null);
+
+export async function initializeRequestNetwork(): Promise<void> {
+  try {
+    const walletClient = createWalletClient({
+      chain: sepolia,
+      transport: custom(window.ethereum!),
```

Comment:
So, as written, it's only compatible with injected wallets?  I'd like to support other wallets too, eventually.